packages/mds-schema-validators/validators.ts

```diff
@@ -75,6 +83,122 @@ const telemetrySchema = Joi.object().keys({
   timestamp: timestampSchema.required()
 })
 
+const ruleSchema = Joi.object().keys({
+  name: Joi.string().required(),
+  rule_id: Joi.string()
+    .guid()
+    .required(),
+  rule_type: Joi.string()
+    .valid(['count', 'time', 'speed', 'user'])
```

Comment:
Please import  `RULE_TYPES` from `mds-types` and have it be `RULE_TYPES.count` instead of `count` in there, so we make sure the rule types align with the type definition. I know it wasn't your original code but this should be corrected. Also the `RULE_TYPES` Enum in `mds-types` should be updated to include the `user` type.

Reply:
I didn't see a count in there anywhere, so I just replaced the array with the constant.

Reply:
OK, so making that change broke several other tests and I don't know that I want to take on tracking all that down just yet.